tests/stub/authorization.py

```diff
@@ -290,7 +291,8 @@ def test_should_fail_with_auth_expired_on_run_using_tx_run(self):
         session = driver.session('r', database=self.get_db())
         tx = session.beginTransaction()
         try:
-            tx.run("RETURN 1 as n").consume()
+            result = tx.run("RETURN 1 as n")
+            result.consume()
```

Comment:
Arguably out of scope of these tests, but do (and do we want to allow it) some drivers  keep "RUN" in their buffer until the result is consumed?

Reply:
I have only updated this to be consistent with your previous change suggestion. It should be identical to what it was before.
In general, I think some drivers (java and probably dotnet) require result to be consumed to display the error.